### ISSUE TITLE
commissioning of 1st board

### ISSUE THREAD

**Issue by JochiSt** [JochiSt/OpenFlowMeter#2]
## Logbook of the commissioning of the first board

**Comment by JochiSt** [comment]
### LEDs
The LEDs work as expected.

They are turned on via:
```
  HAL_GPIO_WritePin(LED4_GPIO_Port, LED4_Pin, GPIO_PIN_RESET);
```
and switched off by:
```
  HAL_GPIO_WritePin(LED4_GPIO_Port, LED4_Pin, GPIO_PIN_SET);
```

**Comment by JochiSt** [comment]
## LEDs
The LEDs are visible, but rather dim. So decreasing the resistance from 2k2 to 1k2 might be a good idea in order to increase visibility.

D1 = red
D2 = yellow
D3 = yellow
D4 = green

**Comment by JochiSt** [comment]
## USART

The USART is working at 115200.

**Comment by JochiSt** [comment]
## CAN Bus

CAN Bus working, both RX and TX. 

Note:       
for RX the CAN filter must be set. There are two FIFOs, which can be handled via an interrupt.

**Comment by JochiSt** [comment]
## ADC
Getting the timer into operation.     
Channel 4: Output Compare No Output
Mode: Toggle on Output

ADC is triggered on Timer 4 Capture Compare 4

Quite helpful:      
* https://www.bartslinger.com/stm32/stm32-cubemx-timer-adc-dma-configuration/

**Comment by JochiSt** [comment]
## PWM 

Works 512 = 1.65V 1024 = 3.3V

**Comment by JochiSt** [comment]
ADC is now running continuously, and gives about 16 000 samples per second of all inputs.

**Comment by JochiSt** [comment]
First board was quite successful, some issues found, which are going to be fixed in revision 2.0. See #11 for commissioning of the new revision.
